### ISSUE TITLE
Setting focus on toolbox pane using keyboard only -- NO_WORKY

### ISSUE THREAD

**Issue by dreftymac** [Komodo/KomodoEdit#2233]
### Context

* KOIDE10
* Windows 7
![capture-001](https://cloud.githubusercontent.com/assets/4074354/21324849/556cb43a-c5d8-11e6-8b4d-697369a07558.png)

### Short Summary

* PROBLEM: Komodo does not set focus to the toolbox pane as advertised in commando command invocation

### Steps to Reproduce

* start komodo
* open any text file
* choose ctrl+shft+l to open the toolbox pane
* invoke commando command: User Interface: Focus on Toolbox Pane

### Expected results

* focus shifts from edit buffer to toolbox pane, with user ability to up-down arrow or type in the filter tools ... input box

### Actual results

* focus remains in the edit buffer

### Additional Information

* ![capture-002](https://cloud.githubusercontent.com/assets/4074354/21324985/e6593ad6-c5d8-11e6-9496-a8dc34cf62f3.png)

**Comment by Defman21** [comment]
Hide your serial number, please.

**Comment by dreftymac** [comment]
Already got it.

**Comment by Defman21** [comment]
Thanks.

**Comment by cgchoffman** [comment]
@Naatan, you're able to repro this?  I use this constantly, it's definitely not broken for me.

**Comment by Naatan** [comment]
Yes, focus remains on editor for me. Are you sure it actually puts the focus on the tools pane for you? ie. pressing arrow keys or typing affects the tools pane, no the editor?

**Comment by cgchoffman** [comment]
Oh my bad, no it doesn't FOCUS.  My bad.  @dreftymac you should be able to perform basically the same action through Commando's Tools scope.  You can select any toolbox item by name through commando.

**Comment by dreftymac** [comment]
**// You can select any toolbox item by name through commando //**

@cgchoffman True, however is that a complete substitute for using the Ko toolbox?

Unless something has been overlooked here, commando is great if you want to **invoke** any arbitrary toolbox item, but it is limited if you want to **manage** any arbitrary toolbox item.

For example, suppose a user wishes to navigate to and modify a snippet, using only the keyboard (no mouse).

If the bug reported here did not exist, the user would have a keyboard-only methodology. 

## Windows recipe

* open the toolbox if not already open
* use commando to set focus on the toolbox
* use the arrow keys plus text filter box to narrow down to desired item
* use the context-menu-key to popup the context menu for the desired item
* choose Properties to get an editable dialog box
* make changes and press OK

**Comment by cgchoffman** [comment]
> @cgchoffman True, however is that a complete substitute for using the Ko toolbox?

No.  But if you're use case is being able to manage the toolbox items then that could be a helpful enhancement request.  I could see "Open Properties" as an additional option on a toolbox item in Commando being helpful.
